frontend/packages/ceph-storage-plugin/integration-tests-cypress/tests/multiple-storageclass-selection.spec.ts

```diff
@@ -44,12 +51,30 @@ describe('Add capacity using multiple storage classes', () => {
     cy.install();
     cy.exec(`echo '${JSON.stringify(testEbsSC)}' | kubectl apply -f -`);
     cy.exec(`echo '${JSON.stringify(testNoProvisionerSC)}' | kubectl apply -f -`);
+
+    cy.exec('oc get nodes -o json').then((res) => {
+      const nodes = JSON.parse(res.stdout);
+      const allWorkerNodesReady = nodes.items.filter((node) => getNodeRole(node) === 'worker');
```

Comment:
```suggestion
      const allWorkerNodes = nodes.items.filter((node) => getNodeRole(node) === 'worker');
```

Reply:
yeah... I copied it from other file and missed correcting it... thanks...